auto_import.sh

```diff
@@ -81,13 +82,38 @@ for dir in $(sudo -u $user find $folder -mindepth 1 -maxdepth 1 -type d -mmin +6
             echo "(that does NOT mean that all files in your SPREADSHEET were imported; it is your" && \
             echo -e "responsibility to check all files from your submission were present in the folder.)\n" )
         fi
+        echo "logfile should be at $logfile"
+        if sudo -u $user [ -f "$logfile" ]; then
+            echo "I have found the logfile"
```

Comment:
more debugging? Or does this go into our logs?

Reply:
Same as above - it goes into our internal logs, not to user logs.